### ISSUE TITLE
rpn_probs and rpn_class_logits foreground vs background

### ISSUE THREAD

**Issue by gaklein2007** [matterport/Mask_RCNN#870]
A quick question about the rpn_probs and rpn_class_logits. In the rpn_graph function when rpn_probs is first defined it is stated that the last dimension is separated into BG/FG. However, when using rpn_class_logits in the rpn_class_loss_graph function, it's last dimension is defined to be FG/BG, which is the reverse of rpn_probs. Is there a reason why it is swapped?

**Comment by keineahnung2345** [comment]
In rpn_class_loss_graph, BG/FG's label are -1/+1 and then 0/1. So the comment in rpn_class_loss_graph should be a typo.
